startup.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+say() {
+	/bin/echo -e "${@}"
+}
+
+err() {
+	say "ERROR: ${@}" 1>&2
+}
+
+fail() {
+	err "${@}"
+	exit 1
+}
+
+PARAMS=()
+
+# -ip string
+#       Grafana IP and port. (default "localhost:3000")
+if [ -n "${GRAFANA_URL}" ] ; then
+	# TODO: Validate the host:port syntax?
+	PARAMS+=("-ip" "${GRAFANA_URL}")
+fi
+
+# -grid-layout
+#       Enable grid layout (-grid-layout=1). Panel width and height will be calculated based off Grafana gridPos width and height.
+[ -n "${GRID_LAYOUT}" ] && PARAMS+=("-grid-layout=${GRID_LAYOUT}")
+
+# -templates string
+#       Directory for custom TeX templates. (default "templates/")
+PARAMS+=("-templates" "/templates")
+
+# -proto string
+#       Grafana Protocol. Change to 'https://' if Grafana is using https. Reporter will still serve http. (default "http://")
+GRAFANA_SSL="${GRAFANA_SSL,,}"
+case "${GRAFANA_SSL}" in
+	1 | y | yes | true | t | on ) GRAFANA_SSL="true" ;;
+	0 | n | no | false | f | off ) GRAFANA_SSL="false" ;;
+	* ) GRAFANA_SSL="false" ;;
+esac
+SSL_PROTO="http://"
```

Comment:
The variable name is confusing, as the protocol could be either http or https. Maybe just `PROTO`? Also, is it really useful to do all that stuff given that SSL will be performed by Istio?

Reply:
Yes, this can probably be improved...

Reply:
@fabricetriboix I remember now why I chose this name: so it'd be consistent with other settings used for other Grafana images when declaring ENV values at instantiation time.

Reply:
Being consistent is good, and being readable is good. It looks like there is a balance to be found here, so I'll just trust you to make the right decision.

Reply:
Yes. I found a 2nd setting that also needed fixing - SSL_CHECK - so now they're both named "GRAFANA_SSL" and "GRAFANA_SSL_CHECK". This makes them easy to identify in environment settings.